publiccode.yml

```diff
@@ -0,0 +1,551 @@
+publiccodeYmlVersion: "0.4"
+
+name: Decidim
+url: "https://github.com/decidim/decidim.git"
+platforms:
+  - web
+developmentStatus: stable
+softwareVersion: "0.29.1"
+releaseDate: "2024-10-02"
+logo: "https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/logo.svg"
+softwareType: "standalone/web"
+
+categories:
```

Comment:
i am not 100% sure if would be spamming, but i guess we can add additional [tags](https://yml.publiccode.tools/categories-list.html#valid-tags) here : `budgeting`, `digital-citizenship`, `regulations-and-directives`, `taxes-management`

Reply:
> budgeting, digital-citizenship, regulations-and-directives.

I added those three